Code/Websites/DanpheEMR/Controllers/Appointment/AppointmentController.cs

```diff
@@ -38,6 +38,8 @@ namespace DanpheEMR.Controllers
 
     public class AppointmentController : CommonController
     {
+        //comment added for testing of git branches: sud-6Feb2023
```

Comment:
this line is not required so please remove this

Reply:
Removed the mentioned line.